wafw00f/plugins/astra.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+
+NAME = '360WangZhanBao (360 Technologies)'
```

Comment:
it seems to me (didn't test yet) that this may overwrite the `wafw00f/plugins/wzb360.py` check because it has the same name. This is how the _plugins_ are loaded:

```python
    for plugin_module in plugin_dict.values():
        wafdetections[plugin_module.NAME] = plugin_module.is_waf
```

Was this your intention? Or do you simply need to change the name?

Reply:
Haha, somehow missed it. This PR is a work in progress work in phase 1. I will review everything during phase 2. These are some small bugs which creep into my code always. My code is spaghetti. :(